From 29a0a637296ca8fb19f442e7c4b5b6e18054a8cd Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Tue, 19 Aug 2014 19:30:54 -0400
Subject: [scsi] qla4xxx: Fix smatch warning in func qla4xxx_conn_get_param

Message-id: <1408476658-5056-17-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88357
O-Subject: [PATCH RHEL 7.1 e-stor 16/20] qla4xxx: Fix smatch warning in func qla4xxx_conn_get_param
Bugzilla: 1089349
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Adheer Chandravanshi <adheer.chandravanshi@qlogic.com>

scsi commit id 971d6fff260f49cf79377a8afb8afab0d61e383b

Signed-off-by: Vikas Chaudhary <vikas.chaudhary@qlogic.com>
Reviewed-by: Mike Christie <michaelc@cs.wisc.edu>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla4xxx/ql4_os.c b/drivers/scsi/qla4xxx/ql4_os.c
index 8508eb3..23fe892 100644
--- a/drivers/scsi/qla4xxx/ql4_os.c
+++ b/drivers/scsi/qla4xxx/ql4_os.c
@@ -2879,7 +2879,6 @@ static int qla4xxx_conn_get_param(struct iscsi_cls_conn *cls_conn,
  struct iscsi_conn *conn;
  struct qla_conn *qla_conn;
  struct sockaddr *dst_addr;
- int len = 0;
 
  conn = cls_conn->dd_data;
  qla_conn = conn->dd_data;
@@ -2893,9 +2892,6 @@ static int qla4xxx_conn_get_param(struct iscsi_cls_conn *cls_conn,
  default:
   return iscsi_conn_get_param(cls_conn, param, buf);
  }
-
- return len;
-
 }
 
 int qla4xxx_get_ddb_index(struct scsi_qla_host *ha, uint16_t *ddb_index)
-- 
1.7.1